pythclient/price_feeds.py

```diff
@@ -112,6 +115,54 @@ def to_dict(self, verbose=False, vaa_format=DEFAULT_VAA_ENCODING):
         return result
 
 
+class MerkleUpdate:
```

Comment:
these names are based on the [Accumulator Wire Format document on Notion](https://www.notion.so/pyth-network/Accumulator-wire-format-3478fcc0eca94e768652d4403f36bc70?pvs=4)